frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card.tsx

```diff
@@ -46,12 +53,12 @@ const getNoobaaStorageClassList = (storageClassesList: K8sResourceKind[]): K8sRe
     _.endsWith(_.get(sc, 'provisioner'), 'noobaa.io/obc'),
   );
 
-const getNoobaaObcList = (
-  obcList: K8sResourceKind[],
+const filterNoobaaResources = (
+  resources: K8sResourceKind[],
   noobaaStorageClassList: K8sResourceKind[],
 ): K8sResourceKind[] =>
-  obcList.filter((obc: K8sResourceKind) => {
-    const storageClassName = _.get(obc, 'spec.storageClassName');
+  resources.filter((res: K8sResourceKind) => {
```

Comment:
existing code..but you dont need to specify `K8sResourceKind`
```suggestion
  resources.filter((res) => {
```